pkgs/development/libraries/qt-6/hooks/qtbase-setup-hook.sh

```diff
@@ -35,16 +35,18 @@ else # Only set up Qt once.
     }
     envBuildHostHooks+=(qmakePathHook)
```

Comment:
We should convert other hooks to also use `addEnvHooks` while at it.

Reply:
Maybe in a different change? This one is already way too big for comfort.